core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -411,6 +412,14 @@ public boolean isReady() {
     return stream.isReady();
   }
 
+  @Override
+  public Attributes getAttributes() {
+    if (stream != null) {
+      return stream.getAttributes();
+    }
+    return super.getAttributes();
```

Comment:
Maybe just do Attributes.EMPTY here. Otherwise, it isn't obvious that calling super is a case we don't care much about.